src/main/java/com/amazon/aocagent/models/Context.java

```diff
@@ -17,4 +20,10 @@
   private String githubSha;
   private OTConfig otConfig;
   private ExpectedMetric expectedMetric;
+  private String launchType;
+  private String deploymentMode;
+  private String defaultSecurityGrpId;
```

Comment:
do you think it's better to put these three fields into stack?

Reply:
I move this one later into stack in another PR